schemas/gazette-notice-schema.json

```diff
@@ -0,0 +1,331 @@
+          "description": "The title of the issue",
+          "type": "string"
+        },
+        "edition_id": {
+          "description": "The identifier of the edition of which the issue is a part",
+          "type": [
+            "string",
+            "integer"
+          ]
+        },
+        "page_range": {
+          "description": "The page range of the issue",
+          "type": "string",
+          "pattern": "^[0-9]*(-[0-9]*)?$"
+        },
+        "url": {
+          "description": "The URL of the issue",
+          "type": "string",
+          "format": "uri"
+        },
+        "media_type": {
+          "description": "The media type of the URL resource",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "publication"
+      ]
+    },
+    "document": {
+      "title": "Document",
+      "description": "A section of an issue",
+      "type": "object",
+      "properties": {
+        "page_range": {
+          "description": "The page range of the document",
+          "type": "string",
+          "pattern": "^[0-9]*(-[0-9]*)?$"
+        },
+        "url": {
+          "description": "The URL of the document",
+          "type": "string",
+          "format": "uri"
+        },
+        "media_type": {
+          "description": "The media type of the URL resource",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "url"
+      ]
+    },
+    "classification": {
+      "title": "Classification",
+      "description": "A classification",
+      "type": "object",
+      "properties": {
+        "scheme": {
+          "description": "The scheme of the classification",
+          "type": "string"
+        },
+        "value": {
+          "description": "The classification",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "value"
+      ]
+    },
+    "body": {
+      "title": "Body",
+      "description": "Content",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The content",
+          "type": "string"
+        },
+        "media_type": {
+          "description": "The media type of the content",
+          "type": "string",
+          "enum": [
+            "text/plain",
+            "text/html"
+          ]
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "value"
+      ]
+    },
+    "update_action": {
+      "title": "Update action",
+      "description": "An act of changing an object",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of change",
+          "type": "string",
+          "enum": [
+            "cancellation",
+            "correction"
+          ]
+        },
+        "object": {
+          "description": "The object that was changed",
+          "type": "object"
```

Comment:
I'm thinking perhaps this object may be reused in future schema, so I don't want to make it GazetteNotice specific.